### ISSUE TITLE
Stackage?

### ISSUE THREAD

**Issue by istathar** [alpmestan/probable#3]
You might consider getting **probable** into Stackage nightly; there's `lts-6` coming up soon, and `lts-7` with GHC 8.0 will be along in a few months.

Haven't used this in a while, but I enjoyed it when I did. It was a nice interface on top of **statistics**.

AfC

**Comment by alpmestan** [comment]
True, I have been using it on and off for a while but never thought about putting it on Stackage. I should definitely do that one of these days, thanks!

**Comment by istathar** [comment]
@alpmestan do you want me to close this?

**Comment by alpmestan** [comment]
... It's still not on Stackage. Sorry. Do you still want/need it there? If you still do I'll add it. (I usually don't even bother adding my personal packages to Stackage, as they're not used all that much)
